src/odemis/driver/test/phenom_test.py

```diff
@@ -31,7 +31,6 @@
 import time
 import unittest
 from unittest.case import skip
-
```

Comment:
no need to remove this empty line, could you add it back?

Reply:
Added it back.